src/ctap/mod.rs

```diff
@@ -105,6 +106,63 @@ fn check_pin_auth(hmac_key: &[u8], hmac_contents: &[u8], pin_auth: &[u8]) -> boo
     )
 }
 
+// Decrypts the HMAC secret salt(s) that were encrypted with the shared secret.
+// The credRandom is used as a secret to HMAC those salts.
+// The last step is to re-encrypt the outputs.
+pub fn encrypt_hmac_secret_output(
+    shared_secret: &[u8; 32],
+    salt_enc: Vec<u8>,
```

Comment:
Is there any reason to prefer a Vec here as opposed to a slice? I don't see any Vec-specific functions being used.

Reply:
The reason was that the input is a Vec, so I just wanted to move it into the function. A slice should be good, too.